chromium/browser_type.go

```diff
@@ -142,14 +142,19 @@ func (b *BrowserType) Launch(opts goja.Value) api.Browser {
 
 	flags := prepareFlags(launchOpts, &state.Options)
 
-	finalDirPath, cleanupFunc, err := b.createTempDir(flags["user-data-dir"], logger)
-	if err != nil {
-		k6common.Throw(rt, err)
+	dataDir := &b.storage
```

Comment:
I know I wrote this, and now I'm seeing that it may not be optimal 🤣 What about making the `storage` field on `BrowserType` a pointer and simply doing here:

```
dataDir := b.storage
```

WDYT?

Reply:
Resolved in commit: 15022675046f46849db9e5f3c3ae3f54b1683cfa